pkg/resourcecreator/googlesqlinstance.go

```diff
@@ -42,10 +42,12 @@ func GoogleSqlInstance(app *nais.Application, instance nais.CloudSqlInstance, pr
 			Settings: google_sql_crd.SQLInstanceSettings{
 				AvailabilityType:    availabilityType(instance.HighAvailability),
 				BackupConfiguration: google_sql_crd.SQLInstanceBackupConfiguration{Enabled: true, StartTime: fmt.Sprintf("%02d:00", *instance.AutoBackupHour)},
+				IpConfiguration:	 google_sql_crd.SQLInstanceIpConfiguration{RequireSsl: true},
```

Comment:
Formatting should be fixed, and for clarity please expand to multiline. Maybe do this with line 44 also?